Gemfile.lock

```diff
@@ -95,9 +95,9 @@ GEM
     grape-entity (0.5.1)
       activesupport
       multi_json (>= 1.3.2)
-    grape-swagger (0.23.0)
+    grape-swagger (0.25.3)
```

Comment:
Sure! Sorry, but how might I go about doing that? Would I have to revert all the changes in Gemfile.lock then recommit and resubmit the pull request?